js/fetchAllVisitingCards.js

```diff
@@ -0,0 +1,57 @@
+const userName = localStorage.getItem('userDirectory');
+const templateWithData = document.getElementById('card-with-data');
+const templateWithoutData = document.getElementById('card-without-data');
+const row = document.getElementById('row');
+
+const fetchAllCards = async () => {
+    displayLoading();
+    await axios.post(
+        'fetch-visiting-cards', {
+        folder_name: userName
+    }
+    ).then((response) => {
+        const allData = response.data.data;
+        if (allData.status == 404) {
+            const card = templateWithoutData.content.cloneNode(true);
+            hideLoading();
+            card.querySelector('.error-message').innerText = "No Data Found.";
+            row.append(card);
+            return;
+        } else if (allData.status == 403) {
+            const card = templateWithoutData.content.cloneNode(true);
+            hideLoading();
+            card.querySelector('.error-message').innerText = "Something Went Wrong.";
+            row.append(card);
+            return;
+        }
+
+        hideLoading();
+        allData.forEach(element => {
```

Comment:
Load only 2 records by default. The user has to click on `Load More` for more cards.